staging/src/k8s.io/kube-aggregator/main.go

```diff
@@ -38,7 +38,7 @@ func main() {
 	logs.InitLogs()
 	defer logs.FlushLogs()
 
-	stopCh := genericapiserver.SetupSignalHandler()
+	stopCh := genericapiserver.SetupSignalHandler(false)
```

Comment:
shouldn't this be set to `true`? I think this is a standalone binary.

Reply:
true. Nobody uses it, but yes. Fixing.